Mathlib/Data/Nat/Cast/ImageInt.lean

```diff
@@ -0,0 +1,49 @@
+/-
```

Comment:
I'm not convinced by the file name, as `Int` is ambiguous between `Interval` and `Integer`. Bytes are cheap, what about renaming it to `ImageInterval` to lift the ambiguity? Or just `Interval`, since we're already in `Nat.Cast` anyway so the context is clear?

Reply:
`Interval` is commonly used for files with lemmas about `Finset.Ixx`, so I named it `SetInterval`.